guides/bug_report_templates/action_cable.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  gem "rails"
+  # If you want to test against edge Rails replace the previous line with this:
+  # gem "rails", github: "rails/rails", branch: "main"
+
+  gem "puma"
+  gem "capybara"
+  gem "selenium-webdriver"
+end
+
+require "action_controller/railtie"
+require "action_cable/engine"
+
+class TestApp < Rails::Application
+  config.root = __dir__
+  config.eager_load = false
+
+  config.logger = Logger.new($stdout)
+  Rails.logger  = config.logger
+
+  config.action_cable.cable = { "adapter" => "async" }
+
+  routes.append do
+    get "/test" => "test#index"
+    get "/broadcast" => "test#broadcast"
+  end
+end
+
+class TestController < ActionController::Base
+  include Rails.application.routes.url_helpers
+
+  def index
+    render inline: DATA.read
+  end
+
+  def broadcast
+    ActionCable.server.broadcast "test_channel", { message: "hello" }
+    render plain: "ok"
+  end
+end
+
+class TestChannel < ActionCable::Channel::Base
+  def subscribed
+    stream_from "test_channel"
+  end
+end
+
+Rails.application.initialize!
+
+require "minitest/autorun"
+
+class BugTest < ActionDispatch::SystemTestCase
+  driven_by :selenium, using: :chrome do |driver_options|
+    driver_options.logging_prefs = { browser: "ALL" }
+    driver_options.add_argument("--headless=new")
+  end
+
+  test "sends logs from the server" do
+    # page.driver.browser.logs.get(:browser)
```

Comment:
This is here because at first I was actually trying to test `console.log` but Chrome adds a bunch of line number type stuff, and `[object Object]` or escaped strings would was awkward to assert on, so here we are.